.cypress/integration/2_notebooks.spec.js

```diff
@@ -12,6 +12,8 @@ import {
   SAMPLE_URL,
   SQL_QUERY_TEXT,
   PPL_QUERY_TEXT,
+  NOTEBOOK_TEXT,
```

Comment:
i don't think so many files should have been changed - @pjfitzgibbons can you rebase?